src/colors/colors.stories.html

```diff
@@ -0,0 +1,58 @@
+<h2>Colors for OpenMRS!</h2>
```

Comment:
A "story" is a link in the sidenav of the styleguide documentation site. It consists of a `stories.js` file and a `stories.html` file. Anything in those files is not shipped as part of the styleguide itself, but just as part of the documentation website for the styleguide.